projects/components/src/skeleton/skeleton.component.ts

```diff
@@ -30,9 +30,9 @@ export class SkeletonComponent implements OnChanges {
 
   public containerClass: string[];
 
-  public static skeletonClass: string = 'skeleton';
+  private static readonly SKELETON_CLASS_NAME: string = 'skeleton';
```

Comment:
nit: we usually keep static properties at the top, right after class definition.